### DISCUSSION TITLE
Model not loading on mobile data

### DISCUSSION THREAD

**Discussion by SethiShreya** [google/model-viewer#4574] [Q&A]
I am using the model viewer snippet in my Wordpress website and the model is loading fine as soon as we are at wifi but as we turn that off and turn on mobile data either the model does not load or load but stuck in middle. 
You can check out the model on my website at [https://altrealitylabs.org/](https://altrealitylabs.org/)
The code snippet i used is written below, I would appreciate any help regarding the issue
Thanks in advance

Code Snippet

```
<style>
    #3d-card{
        background-color: transparent;
    }
</style>

<div id="3d-card">
     <model-viewer src="https://raw.githubusercontent.com/SethiShreya/WordPress-website-resource/main/robot/robot_playground.glb"
    poster="https://raw.githubusercontent.com/SethiShreya/WordPress-website-resource/main/robot/placeholder2.png"
    autoplay
    alt="A 3D model"
    shadow-intensity="1"
    scale="1.5 1.5 1.5"
    camera-controls
    touch-action="pan-y" 
    auto-rotate>
    </model-viewer>
       
      </div>
      <script type="module"
          src="https://unpkg.com/@google/model-viewer/dist/model-viewer.min.js">
      </script>
```

**Comment by elalish** [comment]
I don't think this is related to MV itself. I would recommend connecting your device to a laptop and inspecting the JS console for problems. As an aside, I'd also recommend using any service but `unpkg` - we've noticed a lot of outages with it.
